blocks/columns/columns.css

```diff
@@ -317,7 +332,7 @@ div.img-col > video {
   width: 100%;
 }
 
-.columns > div > div:not(.columns-img-col) div:not(.img-col) {
+.columns:not(.no-padding) > div > div:not(.columns-img-col) div:not(.img-col) {
```

Comment:
It seems that the same style was removed by the pull request [0]. 
Hence, I've introduced a new variant named "absolute-no-padding," and you can find the detailed information about it in [1].
Let me know if you think it is safe go with this style, I will remove the "absolute-no-padding" and go with the same.

[0] https://github.com/hlxsites/sunstar/commit/6bee5dd1dee990b441ebd1a616337b4070a508b3#diff-95c2da244c71fe94382a46e48c77292b821711db947818095f7b3842fc42fc44L238-L241
[1] https://github.com/hlxsites/sunstar/pull/319#issuecomment-1805343604

Reply:
@shrotia It's safe to go with this style. Please revert absolute-no-padding style.